### PR TITLE
Add open parent issue button for a sub-issue edit panel

### PR DESCRIPTION
**Pull request by wazsone**
Signed-off-by: Ruslan Bayandinov <wazsone@ya.ru>

# Contribution checklist

## Brief description

Screenshot 2022-10-13 at 21 22 36

Screenshot 2022-10-13 at 21 22 55

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/view-resources/src/components/UpDownNavigator.svelte [hunk 1]
```diff
@@ -21,8 +31,25 @@
     }
   }
 
+  function openParentIssue () {
+    if (parentIssue) {
+      closeTooltip()
+      showPanel(tracker.component.EditIssue, parentIssue._id, parentIssue._class, 'content')
```
Comment (BykhovDenis) [inline plugins/view-resources/src/components/UpDownNavigator.svelte]: We need support all doc types, not only issue

File: plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte [hunk 2]
```diff
@@ -108,6 +109,10 @@
   $: issueId = currentTeam && issue && getIssueId(currentTeam, issue)
   $: canSave = title.trim().length > 0
   $: isDescriptionEmpty = !new DOMParser().parseFromString(description, 'text/html').documentElement.innerText?.trim()
+  $: {
```
Comment (BykhovDenis) [inline plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte]: '$: parent = issue?.$lookup?.attachedTo' why not?)

**Review by BykhovDenis — CHANGES_REQUESTED** [review]

File: plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte (hunk 2 above)
Comment (wazsone) [reply plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte]: @BykhovDenis Thx)
Fixed

File: plugins/view-resources/src/components/UpDownNavigator.svelte (hunk 1 above)
Comment (wazsone) [reply plugins/view-resources/src/components/UpDownNavigator.svelte]: @BykhovDenis Thank you. Fixed

**Comment by haiodo** [comment]
@wazsone  I suppose we discussed to remember all panels opened from one, another non closed, and allow to do back to previous opened panel. In this case it should be done in panel.ts and updated in showPanel, close Panel methods.

**Comment by wazsone** [comment]
@haiodo As far as I recall. We discussed that browser back(or cmd+arrow_back) have that functionality. And this new back button make sense only in a parent/child navigation. 

image

**Review by haiodo — APPROVED** [review]

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**